components/atom/badge/src/index.scss

```diff
@@ -2,42 +2,51 @@
 
 $c-contrast-atom-badge: $c-white !default;
 $bdrs-atom-badge: $bdrs-m !default;
+$fz-atom-badge-l: $fz-m;
```

Comment:
please add !default to all component vars

Reply:
ok!